xtask/src/commands/test.rs

```diff
@@ -28,6 +29,23 @@ impl Test {
             "--no-demo and --with-demo are mutually exclusive",
         );
 
+        // start building tests, while the subservices are spinning up
+        eprintln!("Starting to build tests...");
```

Comment:
Removed useless comment and clarify user log.

```suggestion
        eprintln!("Starting background process to pre-compile the tests...");
```